src/Ratio.php

```diff
@@ -16,7 +12,7 @@ interface Ratio
     /**
      * @param Currency $base
      * @param Currency $minor
-     * @return Numeric
```

Comment:
what's wrong with it?

Reply:
numeric is a name of pseudo type and phpstan warn about it